src/provider/rpc.ts

```diff
@@ -341,7 +353,7 @@ export class RpcProvider implements ProviderInterface {
     blockIdentifier: BlockIdentifier
   ): Promise<RPC.GetTransactionCountResponse> {
     const block = new Block(blockIdentifier);
-    return this.fetchEndpoint('starknet_getBlockTransactionCount', [block.identifier]);
+    return this.fetchEndpoint('starknet_getBlockTransactionCount', { block_id: block.identifier });
```

Comment:
Same thing as before for `block.identifier`, to use like this everywhere or use declare `blockId` variable instead of `block`

Reply:
https://github.com/0xs34n/starknet.js/pull/332/commits/24c24e8060b76f6a300b21c3148a15183b802fcb